### ISSUE TITLE
rangslider.js input type="range" issue accessbility

### ISSUE THREAD

**Issue by Bindhyachal** [andreruffert/rangeslider.js#299]
I am using the rangeslider.js plugin for my video component on the webpage which works on input type="range" gives below error on running pa11y version 4.13.0
![pa11y-error](https://user-images.githubusercontent.com/30178954/36016989-c9bbed68-0d9b-11e8-9cfd-426f8c41d4e3.JPG)

have tried using attibutes like "name,aria-label,aria-labelledBy" and also adding label element before the input but it does not work.

Kindly help me with the solution.
